kojak/__main__.py

```diff
@@ -1,66 +1,39 @@
-import argparse
 from sys import exit
 
+import kojak.cli as cli
```

Comment:
Why do you use as here? A simple import is sufficient.

Reply:
I want to avoid calls by using long namespaces etc...

Reply:
Sorry I don't understand what you mean with an import from you the same result.